components/tensorboard-controller/controllers/tensorboard_controller.go

```diff
@@ -255,6 +255,8 @@ func generateDeployment(tb *tensorboardv1alpha1.Tensorboard, log logr.Logger, r
 							WorkingDir:      "/",
 							Args: []string{
 								"--logdir=" + mountpath,
+								"--reload_interval",
```

Comment:
This would be a good spot for a comment that warns about the S3 costs associated with this interval. That way if this value needs to be changed in the future by someone else they'll see that context.